### PR TITLE
Add details on editing with Eclipse

### PR DESCRIPTION
**Pull request by jonahgraham**
I thought I had seen this info somewhere else at some point, but now I can't find it. I am not sure how I learned to setup and build LSP4J anymore.

Can another LSP4J committer have a look and see if this is a sensible set of steps?

### CODE REVIEW

**Comment by jonahgraham** [comment]
Nate (a potential contributor) did [successfully](https://dev.eclipse.org/mhonarc/lists/lsp4j-dev/msg00027.html) follow these steps and get up and running.

File: Contributing.md [hunk 1]
```diff
@@ -11,4 +11,33 @@ See https://eclipse.org/legal/ECA.php for details
...
+  - The LSP4J build is gradle based
+- Xtend IDE
+  - Much of the code and tests are written in [Xtend](https://www.eclipse.org/xtend/)
+
+2. Fork/Clone the LSP4J repository
+
+3. Import the root of LSP4J as a Gradle project:
+  - File -> Import
+  - Existing Grade Project
+  - Select root of cloned directory as "Project root directory"
```
Comment (kittaakos) [inline Contributing.md]: `Select root of cloned directory as "Project root directory"` -> `Select the root of the cloned directory as "Project root directory"` What do you think?

File: Contributing.md [hunk 2]
```diff
@@ -11,4 +11,33 @@ See https://eclipse.org/legal/ECA.php for details
...
+  - Finish the wizard - this will import all the projects but there will be lots of errors
+
+4. Run the "eclipse assemble" gradle targets.
+  - There is already a launch configuration called "lsp4j-build-gradle", run it.
```
Comment (kittaakos) [inline Contributing.md]: `There is already a launch configuration called "lsp4j-build-gradle", run it.` -> `There is already a launch configuration in the root of the repository, called "lsp4j-build-gradle", run it.`

**Comment by jonahgraham** [comment]
Thanks @kittaakos, I have made the changes.

**Comment by jonahgraham** [comment]
> Unfortunately, after going through the steps, I still ended up with outgoing changes in the .project files due to Gradle/Buildship. And I had to revert a few other generated classes too. But that does nothing with the steps you wrote down.

This is something that has always bothered me a little too, I submitted #199 for this.

**Pull request merged**